extra/mariabackup/xtrabackup.cc

```diff
@@ -4503,14 +4503,21 @@ bool Backup_datasinks::backup_low()
 
 		if (recv_find_max_checkpoint(&max_cp_field) == DB_SUCCESS
 		    && log_sys.log.format != 0) {
-			if (max_cp_field == LOG_CHECKPOINT_1) {
-				log_header_read(max_cp_field);
+			switch (max_cp_field) {
+				case LOG_CHECKPOINT_1:
+					if (log_header_read(max_cp_field)) {
```

Comment:
I think that we indent `case` at the same level as `switch`. That is, the `switch` body needs to be deindented by one level. That should also avoid excessively long lines.

Reply:
sure. Anything else?